sphinxcontrib/openapi/openapi30.py

```diff
@@ -246,6 +248,49 @@ def _httpresource(endpoint, method, properties, convert, render_examples,
             yield '{indent}{line}'.format(**locals())
         yield ''
 
+    securities = properties.get('security', [])
+    for security in securities:
+        yield ('{indent}This resource requires the '
+               'following authentication scheme(s):').format(**locals())
+        yield ''
+        for ref in security.keys():
+            secScheme = components.get('securitySchemes', {}).get(ref, {})
+            if secScheme['type'] == 'http':
+                if secScheme['scheme'] == 'basic':
+                    line = ':security: HTTP Basic'
+                    yield '{indent}{line}'.format(**locals())
+                if secScheme['scheme'] == 'bearer':
+                    line = ':security: HTTP Bearer'
+                    yield '{indent}{line}'.format(**locals())
+                else:
+                    raise Exception(
+                        'Unknown http scheme "%s"' % secScheme['scheme'])
+            elif secScheme['type'] == 'apiKey':
+                key_loc = secScheme['in']
+                yield ('{indent}:security: '
+                       'API key in {key_loc}').format(**locals())
```

Comment:
I think we should also print `secScheme['name']`. Otherwise, it's not clear in which header to look in.